### DISCUSSION TITLE
Error: Unknown builtin plugin "cleanupIDs" specified. "astro-icon": "^0.8.0"

### DISCUSSION THREAD

**Discussion by gangajamuna** [arthelokyo/astrowind#78] [Q&A]
Hello, I'm trying to use the Astro wind theme. When I run dev everything works as expected however when I try to run the build this error is thrown:
  throw new Error(`[astro-icon] Unable to load icon "${name}"!
Error: [astro-icon] Unable to load icon "tabler:info-square"!
Error: Unknown builtin plugin "cleanupIDs" specified.

I'm unable to preview the build and running localhost:3000 results in 404 not found.
When I remove "astro-icon": "^0.8.0" from project it is working.
Can someone explain to me what is happening here and how I might resolve this?
Thank you!

**Comment by prototypa** [comment]
Hello @gangajamuna,

Thanks for reaching out and help with **AstroWind**. I also get the error. I had not detected it, I cloned the project from 0 and the build gives the same error that you indicate.

I was looking at the `astro-icons` issues and it is something that happens to several developers:

https://github.com/natemoo-re/astro-icon/issues/72

For now I applied a temporary solution that they comment in the thread (https://github.com/natemoo-re/astro-icon/issues/72#issuecomment-1369597045). Update **AstroWind** and let me know if now works for you.

Thank you again.

**Reply by gangajamuna** [reply]
Perfect! That works for me too ;) Thank you very much!
